### ISSUE TITLE
Error when onCardLeftScreen not specified + no TS definition

### ISSUE THREAD

**Issue by nwesthoff** [3DJakob/react-tinder-card#5]
react-tinder-card now expects onCardLeftScreen to be defined, and there's no type for it. I think the type should be added (as optional?) to `index.d.ts` and shouldn't throw an error if it's not defined.

```ts
/**
   * Callback that will be executed when a card has left the screen
  */
  onCardLeftScreen?: () => void
```

**Comment by 3DJakob** [comment]
You're absolutely right, thank you! This has been fixed in `ed5caca` and published in `1.1.4`.

**Comment by nwesthoff** [comment]
Appreciate the quick fix! 🥇 
Thank you  👍
